themes/firedoc-theme-notab/assets/js/api-list.js

```diff
@@ -99,7 +103,8 @@ $(document).ready(function () {
         if (keyword) {
           if (r.type !== 'member') {
             r.this.find('.members-list').removeClass('hidden');
-            itemsCont.prepend(r.this);
+            itemToSort.push(r);
+            // itemsCont.prepend(r.this);
```

Comment:
@yorkie Instead of prepend directly, I add them to an array and sort the array before prepend